src/Symfony/Component/Ldap/Adapter/MoveEntryInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Adapter;
+
+use Symfony\Component\Ldap\Entry;
+use Symfony\Component\Ldap\Exception\LdapException;
+use Symfony\Component\Ldap\Exception\NotBoundException;
+
+/**
+ * @deprecated This interface will be deprecated in 5.0, and merged with `EntryManagerInterface`
```

Comment:
maybe we can alreado do that by using `@method` annotations instead, see also #29434 / #28902

Reply:
To be clear, you're effectively suggesting this: https://people.freebsd.org/~kevans/symfony-ldap-rmmoveentry.diff and moving the move() method from annotation into the interface proper at a later date, yeah?

I guess the documentation from the interface would have to move into the ExtLdap/EntryManager in lieu of the current inheritdoc, too.

Reply:
> moving the move() method from annotation into the interface proper at a later date, yeah?

yes, in 5.0

AFAIK `@inheritdoc` does not inherit from `@method`, so yes until 5.0 that's put in EntryManager

Reply:
Revised; thanks!